spec/factories/works.rb

```diff
@@ -5,7 +5,7 @@
   factory :work do
     title { 'Test title' }
     work_type { 'text' }
-    subtype { ['3D model', 'GIS'] }
+    subtype { %w[Article Essay] }
```

Comment:
The idea here was to test something with whitespace. How about `Presentation slides`?

Reply:
Will do.

Reply:
@jcoyne done. take another look?